tests/Api/DataFixtures/ORM/shop/product_bundles.yml

```diff
@@ -0,0 +1,219 @@
+Sylius\Component\Core\Model\ProductTranslation:
+    productTranslation1:
```

Comment:
You cant create a single entry and refer dynamically via current number. Of course, only if Faker's names are sufficient.